Sources/GRInputField/SwiftUI/InputField.swift

```diff
@@ -390,6 +393,12 @@ public extension InputField {
         return modifiedSelf
     }
 
+    func inputFieldStyle(_ appearance: InputFieldAppearance) -> Self {
```

Comment:
I would prefer the modifier to be named `.inputFieldAppearance`, as "`Style`" reminds me of native SwiftUI modifiers, but those work in a different way, with their own Styles and Configurations. Setting the look of the input field globally is also named `defaultAppearance`/`globalAppearance`.